versions.txt

```diff
@@ -0,0 +1,20 @@
+omni_aws=0.1.0
```

Comment:
should we include `omni` here?

Reply:
including omni will some change CMake files as well, so doing it in another commit